### ISSUE TITLE
use case group

### ISSUE THREAD

**Issue by SirkoS** [i-adopt/glossary#8]
~~provide another level of abstraction above use cases (#4).~~
~~used to group different use cases belonging to a similar "topic".~~

A Use Case Group (UCG) is a categorization of the use cases (#4). It is used to group different use cases belonging to similar kind of applications or activities.

**Comment by gwemon** [comment]
I agree with your defintion @SirkoS but I find "topic" could be better replaced by "application" or "activity". "topic" for me is too dangerously close to "domain topic" or "subject area" and may get people confused.
So, if you agree we could have the following: 
A Use Case Group (UCG) is a categorisation of the use cases (#4). It is used to group different use cases belonging to similar kind of applications or activities.

**Comment by gwemon** [comment]
Linked to https://github.com/i-adopt/usecase_analysis/issues/6

**Comment by SirkoS** [comment]
Fine with me. I adjusted the definition in the first post accordingly.
